Foundation-Mission-Structure-and-Strategy-Proposal.md

```diff
@@ -0,0 +1,76 @@
+# Node.js Foundation Mission, Structure and Strategy
+
+_Proposal to the Node.js Foundation TSC and Board, by Rod Vagg_
+
+**Goal #1:** Agree on a basic statement of mission for the Foundation, upon which all execution, by each group within the Foundation, can be tested and justified.
+
+**Goal #2:** Agree on a strategy generation and execution structure, and the relationship between the main bodies within that structure, that can best achieve the mission of the Foundation.
+
+## Foundation Mission Statement
+
+**The Node.js Foundation exists to increase the adoption of Node.js.**
```

Comment:
Coming out of discussion in the TSC meeting:

"The Node.js Foundation exists to increase the adoption of Node.js and increase participation in the Node.js project."